x-pack/functionbeat/provider/gcp/gcp/transformer/transformer.go

```diff
@@ -0,0 +1,76 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package transformer
```

Comment:
I think we can move the functions in this package to the actual gcp package.

Reply:
Should I move the transformer function in aws as well?

Reply:
> Should I move the transformer function in aws as well?

We can do so in another PR. Is there some 'shared' transformation?

Reply:
No, nothing shared. I will address it in a follow-up PR.